dashboard/app/views/levels/_teacher.html.haml

```diff
@@ -54,3 +57,13 @@
                     %span.puzzle-number
                       = @script_level.position
                 %td.name= link_to student.name, build_script_level_path(@script_level, user_id: student.id, section_id: @section.id)
+  :javascript
+    function resizeScrollable() {
+      var newHeight = $('.teacher-panel').height() -
```

Comment:
Actually, interesting, is there a reason this is `height()` versus the others using `outerHeight()`? My conspiracy theory is that that's where the magic pixels are going :ghost: